numpy v2.0 compatibility

When testing with this package as a dependency and numpy 2.0 dev version, I'm getting the following error:

```txt
[rest of log shortened]
/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/cmocean/tools.py:12: in <module>
    _string_types = (str, np.str_, np.unicode_)
/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/numpy/__init__.py:396: in __getattr__
    raise AttributeError(
E   AttributeError: `np.unicode_` was removed in the NumPy 2.0 release. Use `np.str_` instead.
```

Can the `np.unicode_` be removed in these lines?

https://github.com/matplotlib/cmocean/blob/4b627ffb41d6badf8316fb67700fdd5b91597432/cmocean/tools.py#L11-L14

--- a/cmocean/tools.py
+++ b/cmocean/tools.py
@@ -8,10 +8,7 @@ import numpy as np
 import matplotlib as mpl
 
 
-if sys.version_info > (3,):
-    _string_types = (str, np.str_, np.unicode_)
-else:
-    _string_types = (basestring, np.str_, np.unicode_)
+_string_types = (str, np.str_)
 
 
 def print_colormaps(cmaps, N=256, returnrgb=True, savefiles=False):

